services/brig/src/Brig/Provider/API.hs

```diff
@@ -817,6 +855,36 @@ validateServiceKey pem = liftIO $ readPublicKey >>= \pk ->
         (const $ return Nothing)
         (SSL.readPublicKey (LC8.unpack (toByteString pem)) >>= return . Just)
 
+-- | Send team owners an event about a change in the services whitelist for
+-- their team.
+notifyServiceWhitelistUpdate
+    :: TeamId
+    -> UserId        -- ^ The user who caused the whitelist to change
+    -> ConnId        -- ^ Originating connection
+    -> ProviderId
+    -> ServiceId
+    -> Bool          -- ^ True=added, False=removed
+    -> AppIO ()
+notifyServiceWhitelistUpdate tid uid conn pid sid status = do
+    mbOwners <- RPC.getTeamOwners tid
+    case mbOwners of
+        [] -> pure ()
+        x:xs -> do
+            let owners = fmap (view Teams.userId) (list1 x xs)
+            now <- liftIO getCurrentTime
+            let event = Teams.newEvent eventType tid now
+                          & Teams.eventData .~ Just eventData
+            RPC.rawPush (mkEventList event) owners uid Push.RouteAny (Just conn)
```

Comment:
Not sure if I should filter out `uid` from `owners` here (I've seen it done in some places but I don't exactly understand why it's done)